src/index.html

```diff
@@ -1,13 +1,65 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="style.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body>
+    <article
+       data-qa="card"
+       class="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="imac img"
+        class="card__img"
+      >
+      <h1 class="card__title">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h1>
+      <p
+        class="card__code"
+      >
+        Код товара: 195434
+      </p>
+      <div class="card__rating">
+        <div class="stars">
+          <div class="stars__star">
+          </div>
+          <div class="stars__star">
+          </div>
+          <div class="stars__star">
+          </div>
+          <div class="stars__star">
+          </div>
+          <div class="stars__star">
+          </div>
+        </div>
+        <p class="card__responses">
+         Отзывов: 5
+        </p>
+      </div>
+      <div class="card__price">
+        <p class="card__price-text">
+         Цена:
+        </p>
+        <p class="card__price-value">
+         69 999 грн
+        </p>
+      </div>
+      <button
```

Comment:
According to checklist, "Item with the text Buy should be a link.", so here should be <a> tag)